frontend/public/style/_common.scss

```diff
@@ -275,6 +275,24 @@ dl.co-inline {
   display: block;
 }
 
+.co-external-link-with-copy {
+  display: inline-block;
+  padding-right: $co-external-link-padding-right;
+  &__icon {
+    color: var(--pf-global--link--Color);
+    cursor: pointer;
+    &:hover {
+      color: var(--pf-global--Color--300);
+    }
+  }
+  &__externallinkicon {
+    padding-left: var(--pf-global--spacer--xs);
+  }
+  &__copyicon {
+    padding-left: var(--pf-global--spacer--md);
```

Comment:
At the moment the tooltip is not center aligned over the icon:

![image](https://user-images.githubusercontent.com/139310/141469597-aa9c0d96-8dcd-44b3-8b2c-3d81fc0c3c4a.png)

```suggestion
    margin-left: var(--pf-global--spacer--md);
```

Reply:
updated this